zrml/court/src/lib.rs

```diff
@@ -415,19 +533,12 @@ mod pallet {
                 m.as_mut().unwrap().resolved_outcome = Some(resolved_outcome);
             });
 
-            // Calculate required weight
-            // MUST be updated when new market types are added.
-            if let MarketType::Categorical(_) = market.market_type {
-                if let MarketStatus::Reported = market_status {
-                    Ok(0)
-                } else {
-                    Ok(0)
-                }
-            } else if let MarketStatus::Reported = market_status {
-                Ok(0)
-            } else {
-                Ok(0)
-            }
+            Ok(ResolutionCounters {
+                total_accounts,
+                total_asset_accounts,
+                total_categories,
+                total_disputes,
+            })
```

Comment:
There is no `call` method so the pertinent information for benchmarks is stored in this structure for `PM`.

**Overall, the benchmark history is something that still needs some adjustments**